source/run-command/tests/unified/runCursorCommand.json

```diff
@@ -0,0 +1,215 @@
+{
+  "description": "runCursorCommand",
+  "schemaVersion": "1.3",
+  "createEntities": [
+    {
+      "client": {
+        "id": "client",
+        "useMultipleMongoses": false,
+        "observeEvents": [
+          "commandStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "db",
+        "client": "client",
+        "databaseName": "db"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection",
+        "database": "db",
+        "collectionName": "collection"
+      }
+    }
+  ],
+  "initialData": [
+    {
+      "collectionName": "collection",
+      "databaseName": "db",
+      "documents": [
+        {
+          "_id": 1,
+          "x": 11
+        },
+        {
+          "_id": 2,
+          "x": 22
+        },
+        {
+          "_id": 3,
+          "x": 33
+        },
+        {
+          "_id": 4,
+          "x": 44
+        },
+        {
+          "_id": 5,
+          "x": 55
+        }
+      ]
+    }
+  ],
+  "tests": [
+    {
+      "description": "creates a cursor and iterates results from getMores",
```

Comment:
can you update the test case titles to be more reflective of what the expected inputs are? It's not clear what causes the differences in the expected results between this case and the one below it.

Reply:
Sure, will do, I haven't completed the tests yet but I'm starting with verifying the basic unified test runner operations correctly handle this new operation, so perhaps these tests belong in valid-pass of the unified format?

Reply:
moved comment to new location